plugin/src/main/resources/META-INF/fabric8/profiles-default.yml

```diff
@@ -27,6 +27,7 @@
     - fmp-controller
     - fmp-controller-from-configuration
     - fmp-service
+    - fmp-ingress
```

Comment:
I would refrain from adding ingress to default profile since it needs controller setup also

Reply:
removed